### PR TITLE
[devtools-translate] Portuguese - What's New in DevTools (Chrome 105) #3441

### LINKED ISSUE #3441: [devtools-translate] Portuguese - What's New in DevTools (Chrome 105)
Post: https://developer.chrome.com/blog/new-in-devtools-105/
Translated by: August 19

Translator: @alvarocamillont
Reviewers: @khaosdoctor

### PR DESCRIPTION
**Pull request by alvarocamillont**
Fixes #3441

Changes proposed in this pull request:

Add portuguese translations for What's New in DevTools M105

### CODE REVIEW

File: site/pt/blog/new-in-devtools-105/index.md [hunk 1]
```diff
@@ -29,169 +27,186 @@ draft: true
...
-Chromium issue: [1334484](https://crbug.com/1334484)
+Issue relacionada: [1334484](https://crbug.com/1334484)
 
 
 <!-- ## View and edit @scope at rules in the Styles pane {: #scope } -->
-
+## Visualize e edite @scope em regras no painel Estilos {: #scope }
 <!-- You can now view and edit the [CSS `@scope` at-rules](https://drafts.csswg.org/css-cascade-6/#scope-atrule) in the **Styles** pane.  -->
-
+Agora você pode visualizar e editar o [CSS `@scope` at-rules](https://drafts.csswg.org/css-cascade-6/#scope-atrule) no painel **Estilos**.
```
Comment (khaosdoctor) [inline site/pt/blog/new-in-devtools-105/index.md:164]:
```suggestion
Agora você pode visualizar e editar a funcionalidade do CSS [`@scope at-rules`](https://drafts.csswg.org/css-cascade-6/#scope-atrule) no painel **Estilos**.
```

**Review by khaosdoctor — COMMENTED** [review]
Left one comment over there

File: site/pt/blog/new-in-devtools-105/index.md (hunk 1 above)
Comment (alvarocamillont) [reply site/pt/blog/new-in-devtools-105/index.md:164]: Feito

**Comment by alvarocamillont** [comment]
Done!

**Comment by khaosdoctor** [comment]
LGTM :D

**Comment by jecfish** [comment]
Thanks @alvarocamillont and @khaosdoctor! Over to @andreban for review.

**Review by jecfish — APPROVED** [review]

File: site/pt/blog/new-in-devtools-105/index.md [hunk 2]
```diff
@@ -5,18 +5,16 @@ authors:
...
   - new-in-devtools
   - devtools
   - chrome-105
-draft: true
 ---
 
-*Tradução realizada por [Lucas Santos](https://lsantos.dev). Revisão por [Alvaro Camillo Neto](https://www.linkedin.com/in/alvarocamillont/).*
+*Tradução realizada por [Alvaro Camillo Neto](https://www.linkedin.com/in/alvarocamillont/) . Revisão por [Lucas Santos](https://lsantos.dev).*
 
 {% include 'partials/devtools/pt/banner.md' %}
```
Comment (jecfish) [inline site/pt/blog/new-in-devtools-105/index.md:19]:
```suggestion
{% include 'partials/devtools/pt/banner.md' %}
{% YouTube id='bHw_56RiVsg' %}
```

**Pull request merged**